build/bin/vlan-filtering

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+IFS=$'\n'
+sed_expression="^.*: (.*) state .* : .*master (.*) state.*$"
+vlan_range=1-4094
+for port in $(bridge link show | grep -v @); do
+   outbound_name=$(echo $port | sed -r "s/$sed_expression/\1/")
```

Comment:
I think you are going to capture here also `tap` devices, which you should not.

Reply:
Isn't VMIs tap interfaces at different network namespaces ? or you mean VM out from kubevirt ?

Reply:
you are right about `tap`, but for each VMI you'd see here a half of a `veth` pair which you should not touch.

Reply:
But I am filtering out veths since they appear with "@" so we are not seting vlans for them VMIs or normal pods.

Reply:
@dankenigsberg so we are good ? I don't see any change needed since we are not touching veths at all here